### ISSUE TITLE
requested signature is invalid with PathStyle true

### ISSUE THREAD

**Issue by vito** [rlmcpherson/s3gof3r#63]
When using s3gof3r to download or upload to/from a bucket with credentials provided, I get a `403` from AWS:

```
The request signature we calculated does not match the signature you provided. Check your key and signing method.
```

**Comment by rlmcpherson** [comment]
Thanks for submitting this. I can reproduce the issue but I'm not sure yet how it became broken.

**Comment by rlmcpherson** [comment]
Currently, it only works when there is a '.' in the bucket name, so you could try that as a workaround for now. Generally, using virtual-hosted-style urls is recommended over path-style.

**Comment by rlmcpherson** [comment]
This should now be resolved by #81. Closing but please reopen if there is still an issue.
